packages/hebao_v1/contracts/base/MetaTxModule.sol

```diff
@@ -162,7 +204,27 @@ contract MetaTxModule is BaseModule
         )
         internal
         view
-        returns (address[] memory signers);
+        returns (address[] memory signers)
```

Comment:
I'd prefer creating another private method for this and keep extractMetaTxSigners as an abstract method so we do't have to use the following boilerplate code:

```
signers = super.extractMetaTxSigners(wallet, method, data);
        if (signers.length > 0) {
            return signers;
}
```